packages/web3/src/contract/contract.ts

```diff
@@ -1786,3 +1788,20 @@ export async function getContractEventsCurrentCount(contractAddress: Address): P
       throw error
     })
 }
+
+export const getContractIdFromTxId = async (
```

Comment:
Questions:
1) What if we create multiple contracts in the same tx? 
2) if the created contract is a sub-contract, this doesn't apply, right?

Reply:
> What if we create multiple contracts in the same tx?

This function currently only supports creating one contract, not sure if we need to support multiple contracts

> if the created contract is a sub-contract, this doesn't apply, right?

Yes, the contract id of sub-contract is not calculated based on the tx output index

Reply:
Let's add a comment to say this function only works in the simple case where a single non-subcontract is created in the tx.